Masa.Blazor.Pro/Shared/Settings.razor

```diff
@@ -34,13 +41,21 @@
     [Parameter]
     public string PageModel { get; set; } = PageModes.PageTab;
 
+    [Parameter]
+    public string NavigationStyle { get; set; } = NavigationStyles.Flat;
+
     [Parameter]
     public EventCallback<string> PageModelChanged { get; set; }
 
+    [Parameter]
+    public EventCallback<string> NavigationBarStyleChanged { get; set; }
+
     protected override async Task OnInitializedAsync()
     {
         if (PageModelChanged.HasDelegate)
             await PageModelChanged.InvokeAsync(GlobalConfig.PageMode);
+        if (NavigationBarStyleChanged.HasDelegate)
+            await NavigationBarStyleChanged.InvokeAsync(GlobalConfig.NavigationStyle);
     }
```

Comment:
Is this necessary?

Reply:
Yup, read current navigation style from cache when initializing.